### ISSUE TITLE
[A11Y] [iOS] User unable to download code for any template.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#405]
ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

PRE-REQUISITE:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:
1. Navigate to the URL: https://ampstart.com/
2. Navigate to "Templates" link and double tap on it.
3. Navigate to "Download code" button in any of the sections (Eg: ARTICLE,BLOG POST etc) and double tap on it.

ACTUAL RESULTS:
Download fails. User is unable to download the code.
Error: Safari cannot download this file.

EXPECTED RESULTS:
User should be able to download the code successfully.

ATTACHMENT:
Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
After a bit of reading, I think this is a problem with Mobile Safari itself.

See https://discussions.apple.com/thread/6793241?start=0&tstart=0

https://stackoverflow.com/questions/27391962/a-way-around-the-safari-cannot-download-this-file-error-on-ios

Essentially, Apple decided that since iDevices did not have a file manager, they did not need to include support for files other than music and pdfs since the other built-in apps did not support them.

The fix in this case, is to use another browser: https://forums.macrumors.com/threads/how-to-download-zip-files-to-iphone.1546618/

cc @camelburrito let me know if we can close this

**Comment by camelburrito** [comment]
We dont let you download code on mobile screens. If the download buttons are accessible on the phone that is probably going to stay as we don't have a way to toggle aria attrs with css :).
